substrate/bin/utils/chain-spec-builder/Cargo.toml

```diff
@@ -24,8 +24,12 @@ crate-type = ["rlib"]
 ansi_term = "0.12.1"
 clap = { version = "4.4.6", features = ["derive"] }
 rand = "0.8"
+kitchensink-runtime = { version = "3.0.0-dev", path = "../../node/runtime" }
+log = "0.4.17"
 node-cli = { path = "../../node/cli" }
```

Comment:
This should not be part any more of this binary.

Reply:
I left old functionality in the tool. But yeah removing it is good idea.